README.md

```diff
@@ -1,26 +1,23 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/660d94d0066a658ae731/maintainability)](https://codeclimate.com/github/Lambda-School-Labs/get2it-be/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/660d94d0066a658ae731/test_coverage)](https://codeclimate.com/github/Lambda-School-Labs/get2it-be/test_coverage)
 
-🚫 Note: All lines that start with 🚫 are instructions and should be deleted before this is posted to your portfolio. This is intended to be a guideline. Feel free to add your own flare to it.
-
-🚫 The numbers 1️⃣ through 3️⃣ next to each item represent the week that part of the docs needs to be comepleted by.  Make sure to delete the numbers by the end of Labs.
-
-🚫 Each student has a required minimum number of meaningful PRs each week per the rubric.  Contributing to docs does NOT count as a PR to meet your weekly requirements.
-
 # API Documentation
 
-#### 1️⃣ Backend deployed at [🚫name service here]https://get2it.herokuapp.com/api <br>
+#### 1️⃣ Backend deployed at [Heroku]https://get2it.herokuapp.com/api <br>
 
 ## 1️⃣ Getting started
 
-To get the server running locally:
-
-🚫 adjust these scripts to match your project
+## **Local Server Installation**
+For developing and testing purposes, please follow the instructions below to install a version to your local machine.
 
-- Clone this repo
-- **npm install** to install all required dependencies
-- **npm run server** to start the local server
-- **npm test** to start server using testing environment
+Installing
+1. download/clone backend repo
+2. navigate to cloned repo
+3. install dependencies on your console: `npm i`
+4. install knex globally: `npm i -g knex`
+5. construct a copy of the data base in console: `knex migrate:latest`
+6. populate database with dummy/seeded data: `knex seed:run`
+7. run the server: `npm run server`. Server port default is 3300.
 
```

Comment:
They also need to create an .env file and set JWT_SECRET equal to a string otherwise you get an error for missing string.

Reply:
Perhaps we should have them refer to environment variables below